Jenkinsfile

```diff
@@ -0,0 +1,84 @@
+pipeline {
+    agent none
+
+    environment {
+        GITHUB_USER = credentials('aa4ae90b-b992-4fb6-b33b-236a53a26f77')
+    }
+
+    options {
+        // preserve stashes so that jobs can be started at the test stage
+        preserveStashes(buildCount: 5)
+    }
+
+    stages {
+        stage('Pre-build') {
+            parallel {
+                stage('checkpatch') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.centos:7'
+                            dir 'utils/docker'
+                            label 'docker_runner'
+                            additionalBuildArgs  '--build-arg NOBUILD=1 --build-arg UID=$(id -u)'
+                        }
+                    }
+                    steps {
+                        checkPatch user: GITHUB_USER_USR,
+                                   password: GITHUB_USER_PSW
+                    }
+                    post {
+                        always {
+                            archiveArtifacts artifacts: 'pylint.log', allowEmptyArchive: true
+                        }
+                    }
+                }
+            }
+        }
+        stage('Build') {
+            failFast true
+            parallel {
+                stage('Build on CentOS 7') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.centos:7'
+                            dir 'utils/docker'
+                            label 'docker_runner'
+                            additionalBuildArgs  '--build-arg NOBUILD=1 --build-arg UID=$(id -u)'
+                        }
+                    }
+                    steps {
+                        sconsBuild()
+                        stash name: 'CentOS-install', includes: 'install/**'
+                        stash name: 'CentOS-build-files', includes: '.build_vars-Linux.*, cart-linux.conf, .sconsign-Linux.dblite, .sconf-temp-Linux/**'
+                    }
+                }
+            }
+        }
+        stage('Test') {
+            parallel {
+                stage('Centos Test') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.centos:7'
+                            dir 'utils/docker'
+                            label 'docker_runner'
+                            additionalBuildArgs  '--build-arg NOBUILD=1 --build-arg UID=$(id -u)'
+                        }
+                    }
+                    steps {
+                        runTest stashes: [ 'CentOS-install', 'CentOS-build-files' ],
+                                script: '''bash utils/run_test.sh'''
```

Comment:
You need to add a `junit_files:` parameter to `runTest` such as [this](https://github.com/daos-stack/daos/pull/13/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R137).